redash/query_runner/databricks.py

```diff
@@ -144,10 +144,33 @@ def get_databases(self):
         first_column_name = results["columns"][0]["name"]
         return [row[first_column_name] for row in results["rows"]]
 
-    def get_database_schema(self, database_name):
+    def get_database_tables(self, database_name):
         schema = {}
         cursor = self._get_cursor()
 
+        cursor.tables(schema=database_name)
+
+        for table in cursor:
+            table_name = "{}.{}".format(table[1], table[2])
+
+            if table_name not in schema:
+                schema[table_name] = {"name": table_name, "columns": []}
+
+        return list(schema.values())
+
+    def get_database_tables_with_columns(self, database_name):
+        schema = {}
+        cursor = self._get_cursor()
+
+        # load tables first, otherwise tables without columns are not showed
+        cursor.tables(schema=database_name)
+
+        for table in cursor:
+            table_name = "{}.{}".format(table[1], table[2])
+
+            if table_name not in schema:
+                schema[table_name] = {"name": table_name, "columns": []}
```

Comment:
Had to add a `cursor.tables` here too (previously it was only with `cursor.columns`), otherwise it doesn't return Tables without columns 😬 (causing inconsistency between the results)